go-controller/pkg/ovn/gress_policy.go

```diff
@@ -102,12 +103,16 @@ func (gp *gressPolicy) addPeerAddressSets(asHashNameV4, asHashNameV6 string) {
 // If the port is not specified, it implies all ports for that protocol
 func (gp *gressPolicy) addPortPolicy(portJSON *knet.NetworkPolicyPort) {
 	var pp *libovsdbutil.NetworkPolicyPort
+	protocol := v1.ProtocolTCP
```

Comment:
so it's always gotta be TCP?

Reply:
yeah kube defaults it to TCP for network policy for us in the API, but multinetwork policy does not